src/qt/bitcoinamountfield.cpp

```diff
@@ -79,10 +79,19 @@ class AmountLineEdit: public QLineEdit
 
     void fixup(const QString &input)
     {
-        bool valid = false;
```

Comment:
I think concept NACK on 14608. Afterall, we don't have full blocks, it's still valuable for people to easily send with minimum fee...